### DISCUSSION TITLE
Wire payment module - Reservation period option as select fieldtype

### DISCUSSION THREAD

**Discussion by Caleydon** [PrestaShop/PrestaShop#37052] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

To eliminate user errors, it is a good idea to replace the plain text field for entering the number of days of the reservation with a Select field type with values e.g. ​​from 0 to 30.

### Expected behavior

A value of 0 would turn off the feature in the module. Then you can also rid of the mechanism from the module to check the content of the field and notify the user of the error. Make the module robust to prevent potential errors.

### Steps to reproduce

![reservation-period-field-type](https://github.com/user-attachments/assets/f28d8500-7e1a-44c5-8543-79a82fd6cdf3)

### PrestaShop version(s) where the bug happened

8.2.0

### If your bug is related to a module, specify its name and its version

ps_wirepayment v2.2.0

**Comment by Hlavtox** [comment]
Hi @Caleydon, I converted this issue to the Idea since it fits better here.
